Gruntfile.js

```diff
@@ -69,7 +69,7 @@ module.exports = (grunt) => {
         options: {
           reporter: 'spec',
           require: './source/utils/winston.js',
-          timeout: 15000,
+          timeout: 35000,
```

Comment:
puppeteer default timeout is 30000ms.

Reply:
Would the default be enough?

Reply:
Yes, I also could change the timeout to 15000 which we used before, but opted for sticking with the puppeteer defaults, that's why I have increased it here.

Reply:
Ah, this is the mocha timeout.

I thought this is setting the puppeteer timeout to 35s instead of 30s and was wondering whether not setting it would be fine.